libs/contract/src/lib/contract/+state/contract.service.ts

```diff
@@ -48,4 +50,29 @@ export class ContractService extends CollectionService<ContractState> {
       map(negotiations => negotiations[0])
     );
   }
+
+
+  async create(contractId: string, contract: Partial<Negotiation>) {
+    const activeOrgId = this.orgQuery.getActiveId();
+    const doc = this.getRef().doc(contractId).collection('negotiations').doc()
+    const write = this.batch();
+
+    write.set(doc, {
+      _meta: createDocumentMeta({ createdAt: new Date(), }),
+      status: 'pending',
+      createdByOrg: activeOrgId,
+      sellerId: centralOrgId.catalog,
+      stakeholders: contract.stakeholders,
+      buyerId: contract.orgId,
+      price: contract.price,
+      titleId: contract.titleId,
+      terms: contract.terms,
+      holdbacks: contract.holdbacks,
+      parentTermId: contract.parentTermId,
+      specificity: contract.specificity,
+      orgId: contract.orgId,
+    })
+    this.update(contractId, { status: 'negotiating' }, { write })
```

Comment:
Maybe update only if sale's status is pending :
```suggestion
    const sale = await this.valueChanges(contractId).pipe(take(1)).toPromise(); 
    if (sale.status === 'pending') this.update(contractId, { status: 'negotiating' }, { write });
```